### DISCUSSION TITLE
Language update Dutch 0.54.0

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#780] [General]
Hi,

Language update with the new strings added.

Kind regards,
Henk - https://fileserver.ph5hp.nl - https://ph5hp.nl

[hfs-lang-nl 0-54-0.zip](https://github.com/user-attachments/files/17421541/hfs-lang-nl.0-54-0.zip)

**Comment by rejetto** [comment]
thank you very much!
included in next release.
your file contains a couple of errors, that are fixed here https://raw.githubusercontent.com/rejetto/hfs/refs/heads/54/src/langs/hfs-lang-nl.json

**Reply by PH5HP** [reply]
Hi,

I replaced my version with this corrected version. No idea what was wrong with mine.

Kind regards,
Henk - https://fileserver.ph5hp.nl/ - https://ph5hp.nl/
